src/config.js

```diff
@@ -138,6 +138,12 @@ function parseArgs(argv, config) {
         "Remove cached HTTP responses older than <cache-ttl> seconds old. " +
         "Passing 0 clears and disables cache completely",
     })
+    .option("format", {
+      type: "string",
+      choices: ["text", "json"],
+      default: "text",
+      describe: "Output format for validation results",
+    })
```

Comment:
TODO:

- add this to config file/schema
- is "format" the best name?

Reply:
I'm also going to throw in `--reporter` in the mix here.

Naming thing is hard.

Reply:
I think I'm going to stick with `--format`

Reply:
Silly Chris-from-the-past
https://github.com/chris48s/v8r/issues/494